config/routes.rb

```diff
@@ -18,4 +18,12 @@
 
   resources :cart_items, only: [:create, :destroy]
   get "/cart", to: "cart_items#index"
+
+  get 'errors/file_not_found'
+  get 'errors/unprocessable'
+  get 'errors/internal_server_error'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.